### ISSUE TITLE
Configuration Permissions not working correctly on version 1.4.2

### ISSUE THREAD

**Issue by jlachancekffein** [verbb/navigation#218]
Wow. Now I understand. The settings are duplicate. Here :

![image](https://user-images.githubusercontent.com/24896469/98822425-d65cbf80-23fe-11eb-945d-d49773e28e7c.png)

```<input type="hidden" name="permissions[craft\elements\Entry][enabled]" value="">``` is here two time.

So the last one is always empty.

_Originally posted by @jlachancekffein in https://github.com/verbb/navigation/issues/217#issuecomment-725448955_

**Comment by jlachancekffein** [comment]
Like you see, in the navigation management panel, the entries box is here twice :

![image](https://user-images.githubusercontent.com/24896469/98824605-80d5e200-2401-11eb-969a-e7fb59a4948c.png)

**Comment by engram-design** [comment]
Ah, that's what was going on! I think I know the issue here, and it's due to a non-english language being used, oversight on my part.

Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.3](https://github.com/verbb/navigation/releases/tag/1.4.3)
